Add community page (#176)
FreeTubeApp/FreeTube-Docs: community/community.md

has_children: true

> permalink: /community/

Reviewer: Everything looks good to me, only thing remaining is this

> permalink: /community/

Reviewer 2: Oh you mean the content added?
Ya I agree, all other top level pages have links to child-pages only. I don't see a need to make an exception here.

> permalink: /community/

Reviewer: Some pointers for when you are going to start moving this into its own page:

- Needs to be adjusted to new page name https://github.com/FreeTubeApp/FreeTube-Docs/pull/176/files#diff-4dae2430228d88e8482d74b816f47b8a5340f421bc4a087517d40ec8fb053e51R27
- Can be deleted because its not applicable anymore because its sitting in a child page https://github.com/FreeTubeApp/FreeTube-Docs/pull/176/files#diff-17a8048f75dd986dbf7f0348351ee3cb3f007792811cd6a78ac775dfaacd64e1R8-R11

> permalink: /community/

Author: I will make the requested changes as soon as possible, but I'm also quite busy with life at the moment, so please bare with me here!

Thanks for your input @PikachuEXE  & @efb4f5ff-1298-471a-8973-3d47447115dc!

> permalink: /community/

Author: Just committed the change, sorry for the wait time! Please re-review!

---

## Welcome to the FreeTube community!

> Welcome to the FreeTube community! First of all, we want to say thank you for getting involved in the project!

Author: Thanks, it was not my intention to have it twice, I just forgot to remove it from the text when moving it up to a headline!

> On this page, we want to give you some guidlines on how to behave when contributing to or interacitng with the FreeTube project, its maintainers and the community around it.

Author: Superseded by https://github.com/FreeTubeApp/FreeTube-Docs/pull/176#discussion_r2098745020

## Rules

- We can change and edit these rules as we need to.

> - English only please. It is harder to communicate with others when a language barrier is in effect.

Author: Thank you! All your suggestions really are great improvements and make the rules more understandable and also help to make them sound more friendly!

- Keep in mind that we are a FLOSS and volunteer-led project. All development in the FreeTube project is being done voluntarily and in the spare time of the contributors. Show respect for their work and please refrain from asking when a feature will be implemented or a bug fixed. As a FLOSS project it is impossible for us to provide detailed timelines and we won't make any promises we can't keep.
- Please act appropriately. Should you really be unsure what this means, [GitHub's community guidlines](https://docs.github.com/en/site-policy/github-terms/github-community-guidelines) might help you.
- Do not use AI/LLMs to create issues, pull requests, comments or any other content you submit to the FreeTube project. It is okay to have a LLM give you feedback on your work and help you improve it, but we will not accept any contributions where LLMs did most of the work.
- If you are ever unsure of something, feel free to contact one of the members of the team via the [Matrix chatroom](matrix.md).

## Moderation policy

> Should you break the rules outlined above, for example by insulting others in the FreeTube community or by spamming issues, we will take appropriate moderation steps.

Reviewer: ```suggestion
If you violate the rules outlined above, such as by insulting others in the FreeTube community or spamming issues, we will take appropriate moderation actions.
```

> First, you will receive a written warning from a member of the FreeTube team. They will ask you to stop doing what you are doing and refrain from doing it again in the future. They might also delete the content for which you received the warning.

Reviewer: ```suggestion
Initially, you will receive a written warning from a member of the FreeTube team. They will ask you to cease the behavior and refrain from repeating it in the future. In some cases, they may also delete the content that led to the warning.
```

> Should you continue your behavior or misbehave in any other way, there will not be a second warning. We will ban you from contributing to the FreeTube project in the future.

Reviewer: ```suggestion
If you persist with the inappropriate behavior or engage in any other form of misconduct, there will be no second warning. You will be permanently banned from contributing to the FreeTube project.
```
